step-templates/trigger-multitenant-deployment.json

```diff
@@ -0,0 +1,75 @@
+    "Octopus.Action.SubstituteInFiles.Enabled": "True",
+    "Octopus.Action.EnabledFeatures": "Octopus.Features.SubstituteInFiles"
+  },
+  "Parameters": [
+    {
+      "Id": "f488f5a8-eafc-4da8-97ea-66cc01dfb39e",
+      "Name": "OctopusAPIurl",
+      "Label": "Octopus API URL",
+      "HelpText": null,
+      "DefaultValue": "#{Octopus.Dashboard.BaseUrl}/api",
+      "DisplaySettings": {}
+    },
+    {
+      "Id": "9a4af3e2-340e-4e2f-9529-c160a491cc41",
+      "Name": "OctopusAPIkey",
+      "Label": "Octopus API Key",
+      "HelpText": null,
+      "DefaultValue": "#{Octopus.API.key}",
+      "DisplaySettings": {}

```

Comment:
This should be a `SensitiveValue` as it's an API key to the Octopus instance

Reply:
Henrik, thanks for your comments. I believe and addressed them all.